svelte.js

```diff
@@ -1,26 +1,27 @@
 module.exports = {
     env: {
         browser: true,
-        es6: true,
         node: true,
+        es2024: true,
         'jest/globals': true
     },
-    plugins: ['es', 'jsdoc', 'svelte3', 'jest'],
+    extends: ['eslint:recommended'],
+    plugins: ['es', 'jsdoc', 'jest'],
     parser: '@babel/eslint-parser',
     parserOptions: {
-        sourceType: 'module',
-        ecmaVersion: 2015
+        sourceType: 'module'
```

Comment:
For tao-deliver-fe I needed a recent `ecmaVersion` here, or it wouldn't parse optional chaining (despite also having `es2024: true`).